assets/app/view/game/round/stock.rb

```diff
@@ -16,6 +16,7 @@ class Stock < Snabberb::Component
 
         needs :selected_corporation, default: nil, store: true
         needs :last_player, default: nil, store: true
+        needs :show_bank, default: false
```

Comment:
what's the point of show_bank?

maybe we should make bank a component?

Reply:
Pretty much copied that attribute from the existing stock market tab, but I think you're right that it doesn't look great repeated three times. I'm getting my head around the project structure but can absolutely try and pull it out into its own component which will then be used in those three places.